affiliated/affiliated_package_review_process.md

```diff
@@ -88,56 +105,67 @@ The review process will be overseen and coordinated by the individuals that are
-email to publish the decision, then also edits the pull request to add the
-results to the review to the JSON file. Once the continuous integration passes,
-the pull request is merged.
-
-If the package is rejected, the decision is posted to the pull request, and the
-pull request is closed.
+(and it is at their discretion to decide how to divide up the work).
+
+1. The editor will start off by finding someone to carry out the main review of the
+   package (hereafter the *reviewer*). In this sense the role of the editor is
+   similar to that of a journal editor (and hence the role name). However this analogy
+   is only partially accurate, because here the editor may add their views to the review,
+   unlike a journal editor who is often strictly impartial on the review itself.
+   The pool of available reviewers will be anyone who has an official role on
+   [the Astropy team](http://www.astropy.org/team.html) or anyone who has participated
+   in the Astropy project enough to be familiar with the project guidelines and requirements.
+   In some cases, it may be necessary to have multiple reviewers depending on the
+   expertise required, and if scientific or algorithmic expertise is needed which
+   is not available in the project. A reviewer can be requested from outside the
+   project at the editor's discretion.
+
+2. The editor sends out an email to the potential reviewer to find someone who is
+   willing to review the package, using the [template](#templatereviewrequest)
+   at the bottom of this document.
+   Emails should only be sent to the next potential reviewer(s) - not
+   many possible reviewers at the same time. The editor should let the potential
+   reviewer know that at this point, that the editor may also be doing a review and
+   may synthesize the results into a single review.
+
+3. Once the reviewer accepts, the editor sends the reviewer an email with detailed
+   review instructions (which includes a link to the
+   [review guidelines](https://github.com/astropy/project/blob/master/affiliated/affiliated_package_review_guidelines.md>)),
+   using the [template](#templatereviewinstructions) at the bottom of this document.
+
+4. The editor then leaves a message on the pull request proposing the affiliated package saying:
+
+   *Thank you for proposing this package as an affiliated package! I'm happy to
+   confirm that your package is now under review and we'll post the results of
+   the review here and on the mailing list.*
+
+5. Once the reviewer has finished, they send their review back by email to the
+   editor, who then does their own review if needed and then writes up a review
+   using the review template, which includes a table with badges
+   ([template](#templatereviewresult) at the bottom of this document).
+   At this point, the editor makes a decision to accept or reject the package
+   based on the results of the review. If no objections or concerns are raised,
```

Comment:
Now that I read this again, "If no objections or concerns are raised" does not quite make sense to me. If they are raised, what happens? Isn't that the same as rejection and the author has a month to resolve them?

Reply:
I suspect that that part of the sentence can go. I don't know why it was there initially.